Hi @Vinclame 

The tests fail on this:
```
--> Executing Yamllint on files found in /home/travis/build/dj-wasabi/ansible-zabbix-proxy/...
    /home/travis/build/dj-wasabi/ansible-zabbix-proxy/tasks/main.yml
      80:1      error    trailing spaces  (trailing-spaces)
```

Please take a look at and solve it.

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -77,7 +77,7 @@
   become: yes
   when: ansible_selinux.status == "enabled"
   tags: selinux
-  
+
 - name: "Configure zabbix-proxy"
   template:
     src: zabbix_proxy.conf.j2

